artifactory/commands/packagemanagerlogin/packagemanagerlogin.go

```diff
@@ -40,20 +69,19 @@ func NewPackageManagerLoginCommand(packageManager project.ProjectType) *PackageM
 	}
 }
 
-// packageManagerToPackageType maps project types to corresponding Artifactory package types (e.g., npm, pypi).
-func packageManagerToPackageType(packageManager project.ProjectType) (string, error) {
-	switch packageManager {
-	case project.Npm, project.Yarn:
-		return repository.Npm, nil
-	case project.Pip, project.Pipenv, project.Poetry:
-		return repository.Pypi, nil
-	case project.Go:
-		return repository.Go, nil
-	case project.Nuget, project.Dotnet:
-		return repository.Nuget, nil
-	default:
-		return "", errorutils.CheckErrorf("unsupported package manager: %s", packageManager)
-	}
+// GetSupportedPackageManagersList returns a sorted list of supported package managers.
+func GetSupportedPackageManagersList() []project.ProjectType {
+	allSupportedPackageManagers := maps.Keys(packageManagerToRepositoryPackageType)
+	// Sort keys based on their natural enum order
+	slices.SortFunc(allSupportedPackageManagers, func(a, b project.ProjectType) int {
+		return int(a) - int(b)
+	})
```

Comment:
```suggestion
	slices.Sort(allSupportedPackageManagers)
```
Also - what is the reason for the sorting?

Reply:
to show the interactive list according to the order I prefer, npm near yarn, docker near podman and etc 
@yahavi